backend/commitments/views.py

```diff
@@ -21,7 +45,7 @@ def get_one(request, id_commitment):
       if commitment is None:
          return JsonResponse({'error': 'Commitment not found'}, status=404)
 
-      if not (commitment.id_client == user.id_client_information or user.role == 'employee'):
+      if not (commitment.id_client == user.id or user.role == 'employee'):
```

Comment:
It will not work it's the same as the module route with : 
- if authenticated_user.role != 'employee' or authenticated_user.id != module.id_client 

AND

- if authenticated_user.role != 'employee' or (authenticated_user.id != module.id_client and authenticated_user.role == 'client'):